src/app/Common/Middleware/TokenAuthenticationMiddleware.cs

```diff
@@ -33,10 +34,10 @@ public async Task Invoke(HttpContext context)
 
             if (accessToken.HasValue())
             {
-                var token = await _tokenService.FindAsync(accessToken);
+                var token = await _tokenService.FindAsync(accessToken, TokenTypeEnum.Access);
                 if (token != null && !token.IsExpired())
                 {
-                    var principal = new Principal(new GenericIdentity(token.UserId), new string[] { });
+                    var principal = new Principal(new GenericIdentity(token.UserId.ToString()), new string[] { });
```

Comment:
Redundant .ToString()
Also I advice to use Array.Empty<string> to avoid unnecessary allocations